### PR TITLE
Issue #83: Define the accuracy of key usage times

### PR DESCRIPTION
**Pull request by mwatson2**

### CODE REVIEW

File: encrypted-media-respec.html [hunk 1]
```diff
@@ -922,8 +922,8 @@
...
           <p>
             A session for which the license and any key(s) it contains SHALL NOT be persisted and for which a <var>record of key usage times</var> SHALL be persisted.
-            The <var>record of key usage times</var> consists of a record of the key IDs of the keys used in the session, the <var>first decryption time</var> for each key and the <var>latest decryption time</var> for each key.
-            A <a def-id="message"></a> of type <a def-id="message-type-license-release"></a> containing a record of key usage will be generated when <a def-id="remove"></a> is called and on every <a def-id="load"></a> call until the record is acknowledged by a response passed to <a def-id="update"></a>. Because the license and keys are not persisted, this record implicitly proves that the keys are no longer available in the session.
+            The <var>record of key usage times</var> consists of a record of the key IDs of the keys used in the session, the <var>first decryption time</var> for each key and the <var>latest decryption time</var> for each key, accurate to <var><a def-id="key-usage-accuracy"></a></var>.
```
Comment (ddorwin) [inline encrypted-media-respec.html]: This text should refer to both times. This is a long sentence already, though, and could probably use some reorganizing as mentioned in #87.

File: encrypted-media-respec.html [hunk 2]
```diff
@@ -922,8 +922,8 @@
...
+            A <a def-id="message"></a> of type <a def-id="message-type-license-release"></a> containing a record of key usage will be generated when <a def-id="remove"></a> is called and on every <a def-id="load"></a> call until the record is acknowledged by a response passed to <a def-id="update"></a>. Because the license and keys are not persisted, this record implicitly proves that the keys are no longer available in the session. The <dfn id="key-usage-accuracy"><var>key usage accuracy</var></dfn> is implementation-independant but SHALL NOT be greater than 60 seconds.
```
Comment (ddorwin) [inline encrypted-media-respec.html]: This doesn't really define "key usage accuracy." It's really self-defining. Something like following would be better:
  `<dfn id="key-usage-accuracy"><var>key usage accuracy</var></dfn> is the degree to which.... It is implementation dependent..."`

Comment (ddorwin) [inline encrypted-media-respec.html]: "implementation-specific"?

Comment (ddorwin) [inline encrypted-media-respec.html]: nit: For consistency and maintainability, make the new sentence a new line.

File: encrypted-media-respec.html [hunk 3]
```diff
@@ -2044,15 +2044,19 @@
...
                                   <li>
                                     <p>
-                                      If the <var>record of key usage</var> does not contain an entry for <var>block key ID</var>, add a new entry for <var>block key ID</var> with <var>first decrypt time</var> set to the current time.
+                                      If the <var>record of key usage</var> does not contain an entry for <var>block key ID</var>, add a new entry for <var>block key ID</var> with <var>first decrypt time</var> set to the current time, accurate to within <a def-id="key-usage-accuracy"></a>.
                                     </p>
                                   </li>
                                   <li>
                                     <p>
-                                      Set the <var>latest decrypt time</var> of the entry for <var>block key ID</var> in the <var>record of key use</var> to the current time.
+                                      Set the <var>latest decrypt time</var> of the entry for <var>block key ID</var> in the <var>record of key use</var> to the current time, accurate to within <a def-id="key-usage-accuracy"></a>.
```
Comment (ddorwin) [inline encrypted-media-respec.html]: Perhaps this should be something like `Ensure the <var>latest decrypt time</var>... is accurate to within <a def-id="key-usage-accuracy"></a> of the current time.`

File: encrypted-media-respec.html [hunk 4]
```diff
@@ -2044,15 +2044,19 @@
...
                                     </p>
                                   </li>
                                 </ol>
+                                <p class="note">
+                                  Implementation are free to optimize the times at which this step is executed provided the recorded key usage times remain
```
Comment (ddorwin) [inline encrypted-media-respec.html]: Perhaps something like "...at which <var>latest decrypt time</var> is set..."

Comment (ddorwin) [inline encrypted-media-respec.html]: nit: s/are free to/MAY/

**Comment by mwatson2** [comment]
Updated after merge of PR #87, please review.

**Comment by ddorwin** [comment]
LGTM

**Comment by mwatson2** [comment]
Merged in 1dab9e5e5f81de974f09f41f300926d8411fc62e

**Pull request closed without merging**